src/main/java/command/Command.java

```diff
@@ -0,0 +1,32 @@
+package command;
+
+import constant.ErrorMessage;
+import exception.ErrorHandler;
+import storage.Storage;
+import list.TaskList;
+import ui.Ui;
+
+public abstract class Command {
+    protected boolean isExist = false;
+
+    public boolean getIsExist() {
+        return this.isExist;
+    }
+
+    protected int pareTaskNumber(String taskNumber) throws ErrorHandler {
```

Comment:
typo parseTaskNumber

Reply:
Hi @chenghuiyuan it is fixed, please check again, thanks!